docs/app/pages/showcase/visualizations/sankey-chart/sankey-chart.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="chart-container">
```

Comment:
Some of the text is not visible on node hover. See the size count in the top right corner:

![image](https://user-images.githubusercontent.com/20795331/72165404-52d9d880-33bf-11ea-931a-47dd0f9895fe.png)

![image](https://user-images.githubusercontent.com/20795331/72165414-59685000-33bf-11ea-9e35-2805559b483c.png)